docs/pages_en/QUICKSTART.md

```diff
@@ -0,0 +1,321 @@
+
+### Docker
+Install Docker. Add a Vault user to the Docker group:
+
+```shell
+usermod -a -G docker vault
+```
+
+### Setting up the project
+
+#### Git repository
+
+Create a regular public Git repository.
+
+#### Bucket
+
+Any S3-compatible bucket will do. It should be publicly available for reading.
+
+{% offtopic title="A note about GCS (Google Cloud Storage)" %}
+To get rid of the `An error occurred (AccessDenied) when calling the CreateMultipartUpload operation: Access denied` error, make sure that the Service Account used to access the bucket belongs to the `Storage Admin` role.
+{% endofftopic %}
+
+### Installing the plugin
+
+Download the trdl plugin by following the instructions in the message of the [selected release](https://github.com/werf/trdl/releases). Copy it to `/etc/vault.d/plugins` or another directory where you normally store plugins.
+
+### Configuring the plugin
+
+Setting up Vault includes specifying the directory where the plugins are stored:
+
+```shell
+plugin_directory = "/etc/vault.d/plugins"
+```
+
+Restart Vault.
+
+Register the plugin in Vault:
+
+```shell
+vault plugin register -sha256=$(sha256sum /etc/vault.d/plugins/vault-plugin-secrets-trdl | awk '{print $1}') secret vault-plugin-secrets-trdl
+```
+
+In our case, the plugin file is called `vault-plugin-secrets-trdl`, and we register it under the same name in Vault. Refer to the [official documentation](https://www.vaultproject.io/docs/commands/plugin/register) to learn more about registering plugins.
+
+Enable the plugin as a `secrets engine` at a specific path in Vault:
+
+```shell
+vault secrets enable -path=trdl-test-project vault-plugin-secrets-trdl
+```
+
+You can enable the same plugin many times; however, you must use a unique path each time. For more information, refer to the [official documentation](https://www.vaultproject.io/docs/commands/secrets/enable).
+
+Now let's configure the trdl plugin itself. We will use the [/configure](/reference/vault_plugin/configure.html#configure-plugin) API method to do this:
+
+```shell
+vault write trdl-test-project/configure @configuration.json
+```
+
+where `configuration.json` has the following contents:
+```json
+{
+  "s3_secret_access_key": "FOO",
+  "s3_access_key_id": "BAR",
+  "s3_bucket_name": "trdl-test-project-tuf",
+  "s3_region": "europe-west1",
+  "s3_endpoint": "https://storage.googleapis.com",
+  "git_repo_url": "https://github.com/werf/trdl-test-project",
+  "required_number_of_verified_signatures_on_commit": 2
+}
+```
+
+> When configuring the plugin, you must specify the minimum number of GPG signatures required for a commit (`required_number_of_verified_signatures_on_commit`). Otherwise, the release system becomes vulnerable because any user with access can affect operations that turn out to be unprotected by the quorum defined during the configuration.
```

Comment:
Второе предложение в оригинале слишком длинное, не сразу въедешь. В переводе его тоже тяжело с первого раза прочитать. Возможно, стоит упростить: «В противном случае система обновлений становится уязвимой: операции контролируются не кворумом, а любым пользователем с доступом».

Reply:
А как насчет: "_Otherwise, the release system becomes vulnerable: any user with access can tamper with it because the system is not protected by a quorum._" ?

Reply:
И @olegznv твой вариант: "_Otherwise, the release system becomes vulnerable: operations are not controlled by the quorum, but by any user with access._"

Reply:
Да, отлично

Reply:
```suggestion
> When configuring the plugin, you must specify the minimum number of GPG signatures required for a commit (`required_number_of_verified_signatures_on_commit`). Otherwise, the release system becomes vulnerable: any user with access can tamper with it because the system is not protected by a quorum.
```